src/components/Contexts/Variables/VariablesProvider.tsx

```diff
@@ -48,7 +48,12 @@ export default function VariablesProvider({
   })
 
   // --------- ODB Token (will be removed) ------------------
-  const [odbToken, setOdbToken] = useState("")
+  const [odbToken, setOdbToken] = useState(
+    localStorage.getItem("odbToken") ?? ""
```

Comment:
I see this will be removed, but [useSyncExternalStore](https://react.dev/reference/react/useSyncExternalStore) is usually a good fit for localstorage without having to use an effect

Reply:
Thank you so much, I haven't used it yet, but this kind of feedback is very useful!